src/templates/actionBar.scss

```diff
@@ -98,7 +98,6 @@ $backwards-step4-duration: $standard-fifth-timing;
     &.action_bar--list {
         align-items: center;
         display: flex;
-        justify-content: center;
```

Comment:
Any potential for style regressions here?

Reply:
I don’t believe I check in a hand of locations that have a tid-bit different set of columns and they are good.

Reply:
Sounds good.  All you can do.  Just thought I'd ask it during a code review. 👍